server/routes/admin/locations.js

```diff
@@ -13,26 +13,26 @@ module.exports = lduService => (router, audited) => {
     '/probation-areas',
     asyncMiddleware(async (req, res) => {
       const probationAreas = await lduService.getAllProbationAreas()
-      res.send(probationAreas)
+      res.render('admin/locations/probationAreas', { probationAreas })
     })
   )
 
   router.get(
     '/probation-areas/:probationAreaCode/local-delivery-units',
     asyncMiddleware(async (req, res) => {
-      const ldus = await lduService.getLdusForProbationArea(req.params.probationAreaCode)
-      res.send(ldus)
+      const ldus = (await lduService.getLdusForProbationArea(req.params.probationAreaCode)) || {}
+      res.render('admin/locations/ldus', { ldus, msg: req.flash('success') })
     })
   )
 
   router.post(
     '/probation-areas/:probationAreaCode/local-delivery-units',
     audited,
     asyncMiddleware(async (req, res) => {
-      const { activeLdus } = req.body
+      const activeLdus = req.body.activeLdus || []
```

Comment:
meh.. this is fine, to have got it working with destructuring you could have:
`const { activeLdus = [] } = req.body`

Reply:
done